Resources/Markdown.md

```diff
@@ -0,0 +1,54 @@
+# Markdown
+
+Markdown was designed to be readable as-is, without looking like it has been marked up with tags or formatting instructions (like HTML). It is a markup language commonly used for writing simply formatted documents without needing to use a GUI to format text, such as using **bold** and *italics*. Markdown allows you to write using an easy-to-read and easy-to-write plain text format.
+
+Documentation on programming projects is often lacking. Markdown is an important language for developers to master because it provides a 
+means to discuss a project and features using a language that all web 
+browsers and integrated development environments (IDEs) can understand, and its use across applications (GitHub, Slack, Notion, etc.).. Far 
+easier than hypertext markup language (HTML) to learn, basic Markdown can be 
+learned in a few minutes. 
+
+## Links / Resources
+### Cheat Sheet
```

Comment:
Adding Markdown before 'Cheat Sheet', so `Markdown Cheat Sheet` may make it more clear and understandable.

Reply:
ahh ok will revise

Reply:
added Markdown before Cheat under new pull request as I'm not owner of the forked repo